server/WifiNodeLCFG.xml

```diff
@@ -0,0 +1,505 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
What does this file do? Please enter some description as a comment inside it. Is this something we should have version controlled?